<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectoryClass</class>
 <widget class="QMainWindow" name="DirectoryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>960</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>960</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Directory</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
font-family: &quot;MiSans&quot;;
background-color white;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>540</width>
      <height>935</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="contactsWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>83</y>
        <width>540</width>
        <height>840</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>538</width>
         <height>838</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>540</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>540</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*
{
font-size: 54px;
}</string>
      </property>
      <property name="text">
       <string>联系人</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Widget">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>70</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{ background-color: transparent;color: #000000; 
  font-weight: bold;
  font-size: 48px;
}
QPushButton:hover
{
color: #808080;
}</string>
      </property>
      <property name="text">
       <string>←</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="infoPage">
     <widget class="QLabel" name="detailed">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>540</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>540</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*
{
font-size: 54px;
}</string>
      </property>
      <property name="text">
       <string>详细信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>540</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>540</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*
{
font-size: 36px;
border-top: 1px solid gray;
}</string>
      </property>
      <property name="text">
       <string>总联系人数：</string>
      </property>
     </widget>
     <widget class="QLabel" name="versionlabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>540</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>540</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*
{
font-size: 36px;
border-top: 1px solid gray;
}</string>
      </property>
      <property name="text">
       <string>程序版本：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="About_back">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>70</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{ background-color: transparent;color: #000000; 
  font-weight: bold;
  font-size: 48px;
}
QPushButton:hover
{
color: #808080;
}</string>
      </property>
      <property name="text">
       <string>←</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>520</width>
        <height>36</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>520</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>520</width>
        <height>36</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*
{
font-size: 24px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="version">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>520</width>
        <height>36</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>520</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>520</width>
        <height>36</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*
{
font-size: 24px;
}</string>
      </property>
      <property name="text">
       <string>Beta1.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="update">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>540</width>
        <height>70</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>540</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*
{
font-size: 36px;
border-top: 1px solid gray;
}</string>
      </property>
      <property name="text">
       <string>更新地址：</string>
      </property>
     </widget>
     <widget class="QLabel" name="updateUrl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>520</width>
        <height>72</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>520</width>
        <height>72</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>520</width>
        <height>72</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*
{
font-size: 24px;
}</string>
      </property>
      <property name="text">
       <string>https://github.com/terriblepepper
/myDirectory</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="buffer"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="modifyMenu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="addAction"/>
    <addaction name="deleteAction"/>
    <addaction name="modifyAction"/>
   </widget>
   <widget class="QMenu" name="lookMenu">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="search"/>
    <addaction name="totalInfo"/>
   </widget>
   <addaction name="modifyMenu"/>
   <addaction name="lookMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="addAction">
   <property name="text">
    <string>添加联系人</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>删除联系人</string>
   </property>
  </action>
  <action name="modifyAction">
   <property name="text">
    <string>修改联系人信息</string>
   </property>
  </action>
  <action name="search">
   <property name="text">
    <string>搜索联系人</string>
   </property>
  </action>
  <action name="totalInfo">
   <property name="text">
    <string>统计信息</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Directory.qrc"/>
 </resources>
 <connections/>
</ui>
